modules/post/windows/manage/powershell/load_script.rb

```diff
@@ -0,0 +1,48 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+require 'rex'
+
+class Metasploit3 < Msf::Post
+  include Msf::Post::Windows::Powershell
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'                 => "Load Scripts Into PowerShell Session",
+      'Description'          => %q{
+        This module will download and execute one or more PowerShell script
+        s over a present powershell session.
+        Setting VERBOSE to true will show the stager results.
+      },
+      'License'              => MSF_LICENSE,
+      'Platform'             => ['win'],
+      'SessionTypes'         => ['powershell'],
+      'Author'        => [
+          'Ben Turner benpturner[at]yahoo.com',
+          'Dave Hardy davehardy20[at]gmail.com'
+        ]
+    ))
+
+    register_options(
+      [
+        OptPath.new( 'SCRIPT',  [false, 'Path to the PS script', ::File.join(Msf::Config.install_root, "scripts", "ps", "msflag.ps1") ]),
```

Comment:
It was just the same as the one in exec_powershell script so I though I would use that. We could remove the default all together if you prefer